tacacs_plus/client.py

```diff
@@ -32,7 +32,7 @@ class TACACSClient(object):
     http://www.shrubbery.net/tac_plus/
     """
 
-    def __init__(self, host, port, secret, timeout=10, session_id=None,
+    def __init__(self, host, port, secret, timeout=10, session_id=None,client_type="IPv4",
```

Comment:
Instead of taking a special string, could we just take the family as an argument here?

```python
def __init__(self, host, port, ..., family=socket.AF_INET,
```